legend-engine-xt-relationalStore-trino-execution-tests/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/tests/api/dynamicTestConnections/TrinoTestContainersWithDelegatedKerberosAuth.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Goldman Sachs
```

Comment:
does this work, and used. If not, we should delete this file

Reply:
This is getting used as part of the TrinoDelegatedKerberosAuth Test